src/legacy/ui/public/vis/editors/default/components/default_editor_agg.tsx

```diff
@@ -0,0 +1,291 @@
+      actionIcons.push({
+        id: 'dragHandle',
+        type: 'grab',
+        tooltip: i18n.translate('common.ui.vis.editors.agg.modifyPriorityButtonTooltip', {
+          defaultMessage: 'Modify priority by dragging',
+        }),
+        dataTestSubj: 'dragHandleBtn',
+      });
+    }
+    if (isRemovable) {
+      actionIcons.push({
+        id: 'removeDimension',
+        color: 'danger',
+        type: 'cross',
+        onClick: () => removeAgg(agg),
+        ariaLabel: i18n.translate('common.ui.vis.editors.agg.removeDimensionButtonAriaLabel', {
+          defaultMessage: 'Remove dimension',
+        }),
+        tooltip: i18n.translate('common.ui.vis.editors.agg.removeDimensionButtonTooltip', {
+          defaultMessage: 'Remove dimension',
+        }),
+        dataTestSubj: 'removeDimensionBtn',
+      });
+    }
+    return (
+      <div>
+        {actionIcons.map(icon => {
+          if (icon.id === 'dragHandle') {
+            return (
+              <EuiIconTip
+                key={icon.id}
+                type={icon.type}
+                content={icon.tooltip}
+                iconProps={{
+                  ['aria-label']: icon.ariaLabel,
+                  ['data-test-subj']: icon.dataTestSubj,
+                }}
+                position="bottom"
+              />
+            );
+          }
+
+          return (
+            <EuiToolTip key={icon.id} position="bottom" content={icon.tooltip}>
+              <EuiButtonIcon
+                iconType={icon.type}
+                color={icon.color as Color}
+                onClick={icon.onClick}
+                aria-label={icon.ariaLabel}
+                data-test-subj={icon.dataTestSubj}
+              />
+            </EuiToolTip>
+          );
+        })}
+      </div>
+    );
+  };
+
+  const buttonContent = (
+    <EuiFlexGroup gutterSize="xs" alignItems="center">
+      <EuiFlexItem grow={false} className="eui-textTruncate">
+        {agg.schema.title}
+      </EuiFlexItem>
+      <EuiFlexItem className="eui-textTruncate">
+        {showDescription && (
+          <EuiText size="s" color="subdued">
+            <p className="eui-textTruncate">{getDescription()}</p>
+          </EuiText>
+        )}
+        {showError && (
+          <EuiTextColor
+            color="danger"
+            aria-label={i18n.translate('common.ui.vis.editors.agg.errorsAriaLabel', {
+              defaultMessage: 'Aggregation has errors',
+            })}
+          >
+            <h6>
+              <FormattedMessage id="common.ui.vis.editors.agg.errorsText" defaultMessage="Errors" />
+            </h6>
+          </EuiTextColor>
+        )}
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
+  return (
+    <EuiAccordion
+      id={`visEditorAggAccordion${agg.id}`}
+      initialIsOpen={isEditorOpen}
+      buttonContent={buttonContent}
+      className="visEditorSidebar__section visEditorSidebar__collapsible visEditorSidebar__collapsible--marginBottom"
+      paddingSize="s"
```

Comment:
Do we need this padding?
FMPOV it steals space in an agg..

![padding](https://user-images.githubusercontent.com/31325372/61306241-e9ed1b80-a7f4-11e9-8e90-27ac02550e98.png)

Reply:
Thanks. Removed.